lib/config/options/index.ts

```diff
@@ -1929,6 +1929,14 @@ const options: RenovateOptions[] = [
     default: false,
     supportedPlatforms: ['gitlab'],
   },
+  {
+    name: 'forkModeDisallowMaintainerEdits',
+    description:
+      'Disallow maintainers to push to Renovate pull requests when running in fork mode.',
+    type: 'boolean',
+    supportedPlatforms: ['github'],
+    default: true,
```

Comment:
```suggestion
    default: false,
```
so we don't break existing users

Reply:
good catch. I reversed the option with the recent changs